src/components/atoms/picker-sheet/stories/index.stories.web.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
Does this need to be web-only? It should be possible to give this component one story for all platforms